### PR TITLE
Update Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

**Comment by nhirokinet** [comment]
Wow, thank you for very fast translation!

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -62,10 +61,7 @@
         \n-&#032;google &lt;query&gt;
         \n-&#032;wikipedia &lt;query&gt;
         \n-&#032;yahoo &lt;query&gt;
-        \n
```
Comment (nhirokinet) [inline app/src/main/res/values-zh-rCN/strings.xml:65]: Sorry, I added these 3 lines after first adding ping command. Can you include there line?

Comment (sr093906) [reply app/src/main/res/values-zh-rCN/strings.xml:65]: @nhirokinet Should be okay now. Last edit was based on https://github.com/nhirokinet/bluelineconsole/compare/nhirokinet:7c48486...nhirokinet:6a97e1d

**Comment by nhirokinet** [comment]
Thank you !

**Pull request merged**